app/views/votes/_voting.html.erb

```diff
@@ -1,11 +1,10 @@
 <% if @user_votes.include? vote %>
-  <% Vote::VOTE_TYPES.each do |reason, value| %>
+  <% t(:votes).each do |value, reason| %>
     <%= link_to reason, quote_vote_path(quote, vote, { vote: { value: value } } ), method: :patch %>
   <% end %>
   <%= link_to "Erase my vote", quote_vote_path(quote, vote), method: :delete %>
 <% else %>
-  <% Vote::VOTE_TYPES.each do |reason, value| %>
+  <% t(:votes).each do |value, reason| %>
```

Comment:
This wasn't my first idea (or something I was aware of) but it is what @Senjai had recommended as something that was somewhat standard.  Would you recommend a method or constant on the model or a helper method?  Something else entirely?